x-pack/plugins/monitoring/public/alerts/badge.tsx

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Fragment } from 'react';
+import {
+  EuiContextMenu,
+  EuiPopover,
+  EuiBadge,
+  EuiFlexGrid,
+  EuiFlexItem,
+  EuiText,
+} from '@elastic/eui';
+import { CommonAlertStatus, CommonAlertState } from '../../common/types';
+import { AlertSeverity } from '../../common/enums';
+// @ts-ignore
+import { formatDateTimeLocal } from '../../common/formatting';
+import { AlertState } from '../../server/alerts/types';
+import { AlertPanel } from './panel';
+import { Legacy } from '../legacy_shims';
+import { isInSetupMode } from '../lib/setup_mode';
+
+function getDateFromState(states: CommonAlertState[]) {
+  const timestamp = states[0].state.ui.triggeredMS;
+  const tz = Legacy.shims.uiSettings.get('dateFormat:tz');
+  return formatDateTimeLocal(timestamp, false, tz === 'Browser' ? null : tz);
+}
+
+interface Props {
+  alerts: { [alertTypeId: string]: CommonAlertStatus };
+}
+export const AlertsBadge: React.FC<Props> = (props: Props) => {
+  const [showPopover, setShowPopover] = React.useState<AlertSeverity | boolean | null>(null);
+  const inSetupMode = isInSetupMode();
+  const alerts = Object.values(props.alerts).filter(Boolean);
+
+  const badges = [];
+
+  if (inSetupMode) {
+    const button = (
+      <EuiBadge
+        iconType="bell"
+        onClickAriaLabel={`${alerts.length} alert(s)`}
```

Comment:
nit: What do you think about making a global function somewhere to take care of this case automatically? Something like: 
```javascript
const numberOfAlertsLabel = (count: number) => `${count} alert${(count > 1) ? 's' : '' }`;
```
This way we can localize it in one place